ci/tasks/publish.sh

```diff
@@ -9,10 +9,11 @@ for file in $COPY_KEYS ; do
   file="${file/\%s/$VERSION}"
 
   echo "$file"
+  filename=$(${file} | grep -E -o 'bosh-stemcell.*.tgz' )
```

Comment:
I don't believe this works as expected. This would try to execute `${file}`, which doesn't resolve and isn't executable. I think you meant `echo "${file}"`. But, I think it's simpler to do...

    filename=$(basename "$file")